{
  "message": "ValueError: Error when checking model input: the list of Numpy arrays that you are passing to your model is not the size the model expected. Expected to see 1 array(s), but instead got the following list of 2 arrays: [array([[[ 1.0182865 , -0.74200785, -0.70739686, ...,  1.186325  ,\n          1.3607498 ,  0.8599428 ],\n        [ 1.0856249 , -1.3634267 , -0.48535436, ...,  1.0903953 ,\n          1.3607498 ,  1.417257...",
  "traceback": ["py_call_impl(callable, dots$args, dots$keywords)", "(structure(function (...) \n{\n    dots <- py_resolve_dots(list(...))\n    result <- py_call_impl(callable, dots$args, dots$keywords)\n    if (convert) {\n        result <- py_to_r(result)\n        if (is.null(result)) \n            invisible(result)\n        else result\n    }\n    else {\n        result\n    }\n}, class = c(\"python.builtin.method\", \"python.builtin.object\"\n), py_object = <environment>))(x = list(<environment>, <environment>), \n    y = NULL, batch_size = 32L, verbose = 1L, sample_weight = NULL)", "do.call(object$evaluate, args)", "evaluate.keras.engine.training.Model(., test_gen())", "evaluate(., test_gen())", "function_list[[k]](value)", "withVisible(function_list[[k]](value))", "freduce(value, `_function_list`)", "`_fseq`(`_lhs`)", "eval(quote(`_fseq`(`_lhs`)), env, env)", "eval(quote(`_fseq`(`_lhs`)), env, env)", "withVisible(eval(quote(`_fseq`(`_lhs`)), env, env))", "model %>% evaluate(test_gen())", "eval(ei, envir)", "eval(ei, envir)", "tfruns::training_run(\"src/03-B-RNN.R\")"]
}
